### ISSUE TITLE
[AzureRM.Netcore] Import-AzureRmContext fails

### ISSUE THREAD

**Issue by alexandair** [Azure/azure-powershell#4488]
I have mycontext.json file that I've created using the Save-AzureRmContext cmdlet.
When I try to import it with the Import-AzureRmContext, I get:
```
PS /home/aleksandar> import-azurermcontext -Path ./clouddrive/mycontext.json
import-azurermcontext : Exception has been thrown by the target of an invocation.
```

**Comment by sambanks** [comment]
Hi Alexandair

Did you ever find a fix or a workaround for this? We are seeing the same thing.

Cheers

**Comment by cormacpayne** [comment]
@alexandair @sambanks this looks like an issue with Newtonsoft.Json in .NET Core

@twitchax can you confirm this is the issue?

**Comment by sambanks** [comment]
@cormacpayne I gave up on the whole endeavor. I happily no longer have to work with Azure

**Comment by twitchax** [comment]
This is a bug in the version of Newtonsoft.Json that ships with PowerShell 6.0.  The bug has been fixed in Newtsonsoft.Json, but PowerShell 6.0 needs to ingest that fix.  Closing.

**Comment by twitchax** [comment]
Verified the fix in PowerShell 6.0 Beta 9.

```bash
docker run -it --rm twitchax/azure-powershell-core: 4437e724ace572095b9380136c8964a8eb49e1c1
```
